### PR TITLE
[v6r20] PilotsLogging fixes

### PR DESCRIPTION
**Pull request by miloszz**
Initial version of PilotsLogging had some issues, that were discovered during testing.

BEGINRELEASENOTES

*WorkloadManagementSystem
FIX: Accessing PilotsLogging Queue in CS

ENDRELEASENOTES

### CODE REVIEW

File: WorkloadManagementSystem/ConfigTemplate.cfg [hunk 1]
```diff
@@ -88,6 +88,7 @@ Services
       Services =
     }
```
Comment (fstagni) [inline WorkloadManagementSystem/ConfigTemplate.cfg]: I think this `Dependencies` part is useless (and confusing). When was introduced here?

File: WorkloadManagementSystem/Service/PilotsLoggingHandler.py [hunk 2]
```diff
@@ -14,33 +14,34 @@
...
+  def initializeHandler( cls, serviceInfoDict ):
     """Initialization of Pilots Logging service
     """
     cls.consumersSet = set()
     cls.pilotsLoggingDB = PilotsLoggingDB()
-
-    result = createConsumer( cls.srv_getCSOption("PilotsLoggingQueue"), callback = cls.consumingCallback )
+    queue = cls.srv_getCSOption("PilotsLoggingQueue")
+    # This is pretty awful hack. Somehow, for uknown reason, I cannot access CS with srv_getCSOption.
+    # The only way is using full CS path, so I'm using it as a backup solution.
```
Comment (fstagni) [inline WorkloadManagementSystem/Service/PilotsLoggingHandler.py]: Put it in `initialize` method.

Comment (miloszz) [reply WorkloadManagementSystem/Service/PilotsLoggingHandler.py]: `initialize` is called on each call to the service. Here I want to create consumer on service startup.

File: WorkloadManagementSystem/ConfigTemplate.cfg (hunk 1 above)
Comment (fstagni) [reply WorkloadManagementSystem/ConfigTemplate.cfg]: Just remove it.

File: WorkloadManagementSystem/Agent/PilotMonitorAgent.py [hunk 3]
```diff
@@ -33,7 +33,7 @@ def initialize( self ):
     self.clearPilotsDelay = self.am_getOption( 'ClearPilotsDelay', 30 )
     self.clearAbortedDelay = self.am_getOption( 'ClearAbortedPilotsDelay', 7 )
 
-    self.WMSAdministrator = WMSAdministratorHandler()
+    self.WMSAdministrator = RPCClient('WorkloadManagement/WMSAdministrator')
```
Comment (andresailer) [inline WorkloadManagementSystem/Agent/PilotMonitorAgent.py]: Could you please check with pylint if there are errors and add 
e.g., 
`#pylint disable=no-member`
for the clearPilots call

Comment (miloszz) [reply WorkloadManagementSystem/Agent/PilotMonitorAgent.py]: My pylint isn't signalling anything in clearPilots call (similar situation in client using rpc to call service).

Comment (andresailer) [reply WorkloadManagementSystem/Agent/PilotMonitorAgent.py]: Ok, thanks for checking.

**Comment by fstagni** [comment]
Still WIP?

**Comment by miloszz** [comment]
Yes, still some issues to be addressed.

**Pull request merged**